package.json

```diff
@@ -62,7 +62,7 @@
     "react": "0.13.1",
     "react-hammerjs": "0.2.2",
     "react-imageloader": "1.2.0",
-    "url-pattern": "^0.8.0",
+    "url-pattern": "git://github.com/k88hudson/url-pattern.git#hyphenunderscore",
```

Comment:
what does this change and is there already something that does this so we don't have to point to personal git repos?